web_src/js/components/RepoActionView.vue

```diff
@@ -384,8 +393,18 @@ export function ansiLogToHTML(line) {
   transition: transform 0.2s;
 }
 
+.action-view-header .run_rerun {
+  border: none;
+  color: var(--color-green);
+  background-color: transparent;
+  outline: none;
+  cursor: pointer;
+  transition: transform 0.2s;
+}
+
 .action-view-header .run_cancel:hover,
-.action-view-header .run_approve:hover {
+.action-view-header .run_approve:hover,
+.action-view-header .run_rerun:hover {
```

Comment:
Such duplicate doesn't look good to me

<img width="425" alt="image" src="https://user-images.githubusercontent.com/2114189/234518529-528fa460-9d84-4846-8cda-f418ca13cb0b.png">

Reply:
I have changed the code here. Does it look good to you?

Reply:
Thank you very much.